src/main/kotlin/no/nav/familie/ks/sak/api/dto/SanityBegrunnelseMedEndringsårsakResponseDto.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.familie.ks.sak.api.dto
+
+import no.nav.familie.ks.sak.kjerne.brev.begrunnelser.IBegrunnelse
+import no.nav.familie.ks.sak.kjerne.endretutbetaling.domene.Årsak
+
+data class SanityBegrunnelseMedEndringsårsakResponseDto(
+    val id: IBegrunnelse,
+    val navn: String,
+    val endringsårsaker: List<Årsak>,
```

Comment:
Stemmer på en rename av Årsak -> Endringsårsak

Reply:
Jeg er enig at denne kan renames, men er usikker på om det bør gjøres i denne PRen. Det kan føre til endringer i DTOer og lignende som kan gi uante side-effects. Jeg gjør ikke denne endringen i denne PRen i hvert fall, for å holde scopet nede.

Reply:
Endring av enum navn skal forsåvidt ikke gi innvirkninger på DTO navn med mindre man endrer på feltnavnet også i DTO'n :D 
Men ja, ta det i en annen pr.